### DISCUSSION TITLE
`#pragma once` instead of `#ifndef`

### DISCUSSION THREAD

**Discussion by shemeshg** [hsutter/cppfront#1113] [Suggestions]
`#pragma once` instead of

```
#ifndef _VOLUMES_RAM_DISK_4G_CPP2EXAMPLE_BUILD__CPPFRONT_TEST_H_CPP2
#define _VOLUMES_RAM_DISK_4G_CPP2EXAMPLE_BUILD__CPPFRONT_TEST_H_CPP2

...
...

```

**Comment by hsutter** [comment]
Thanks! I considered that, but `#pragma once` is not standard. I want to stick with portable Standard C++ for Cpp1 so that it works with the widest variety of conforming compilers.
